composer/spec/fixtures/composer_files/pear

```diff
@@ -6,6 +6,6 @@
     }
   ],
   "require": {
-    "pear-pear.horde.org/Horde_Date": "^2.4.0@stable"
+    "pear-pear.horde.org/horde_date": "^2.4.0@stable"
   }
 }
```

Comment:
PEAR support is gone in v2

Reply:
Didn't even know that, but haven't used pear in decades. Removed the fixtures for it